### PR TITLE
world-notifications: Adding likes and ignoring others

### PR DESCRIPTION
**Pull request by pazdera**
This change adds support for likes and makes sure no unknown notifications will be shown in the future.

cc @alex5imon

### CODE REVIEW

**Comment by pazdera** [comment]
Related to: https://github.com/KanoComputing/peldins/issues/2200

**Comment by pazdera** [comment]
This is now ready for review.

cc @alex5imon @tombettany @convolu @neue

File: kano_world/session.py [hunk 1]
```diff
@@ -635,6 +596,53 @@ def _process_notification(self, entry):
...
+                n['image'] = ART_SHARE_IMG
+            elif entry['type'] == 'featured':
+                n['title'] = 'Staff picked!'
+                n['image'] = FEATURED_SHARE_IMG
+
+            # Link to the share
+            #share_id = self._get_dict_value(entry, ['meta', 'item', 'id'])
+            #if share_id:
+            #    n['command'] = "kano-world-launcher /shared/{}".format(
+            #        share_id)
```
Comment (tombettany) [inline kano_world/session.py]: Do we want this commented out? Should it be in or should it just be removed?

**Comment by tombettany** [comment]
Minor not but otherwise looks okay to me @pazdera

File: kano_world/session.py (hunk 1 above)
Comment (pazdera) [inline kano_world/session.py]: That certainly shouldn't be there. Good spot!

**Pull request merged**